src/Task/PhpCsFixerV2.php

```diff
@@ -91,6 +86,8 @@ public function run(ContextInterface $context)
             return $this->runOnAllFiles($context, $arguments);
         }
 
+        $arguments->add('--path-mode=intersection');
```

Comment:
you could know the classes and still execute phar from cli.

that new mode is a hacky way of doing this which abuse package that is external for you, putting special mode only for one user is no-go.
and that option is not `intersect_on_commit`, it's more like `intersect_wich_finder_from_config_but_ignore_if_there_is_no_finder_in_config`... hacky